packages/account/src/BiconomySmartAccount.ts

```diff
@@ -1,20 +1,22 @@
 import { JsonRpcProvider } from '@ethersproject/providers'
-import {ethers, BigNumberish, BytesLike, BigNumber } from 'ethers'
+import { ethers, BigNumberish, BytesLike, BigNumber } from 'ethers'
 import { SmartAccount } from './BaseAccount'
 import {
+  Logger,
   NODE_CLIENT_URL,
   RPC_PROVIDER_URLS,
-  EntryPoint_v100__factory,
   SmartAccountFactory_v100,
-  SmartAccountFactory_v100__factory,
-  SmartAccount_v100__factory
+  getEntryPointContractInstance,
+  getFactoryContractInstance,
+  getProxyContractInstance
```

Comment:
Rename getProxyContractInstance to getSAProxyContract

We can remove Instance from all other names as well.

It should be `getEntryPointContract`, `getSAFactoryContract`, `getSAProxyContract`

Reply:
resolved